cli/running/running.go

```diff
@@ -225,7 +238,16 @@ func FillCtx(cliOpts *modules.CliOpts, cmdCtx *cmdcontext.CmdCtx, args []string)
 	cmdCtx.Running.LogMaxSize = cliOpts.App.LogMaxSize
 	cmdCtx.Running.LogMaxAge = cliOpts.App.LogMaxAge
 	cmdCtx.Running.LogMaxBackups = cliOpts.App.LogMaxBackups
-
+	if cliOpts.App.DataDir == "" {
+		curDir, _ := os.Getwd()
+		cmdCtx.Running.DataDir = filepath.Join(curDir, "."+appName+"_data_dir")
+	} else {
+		cmdCtx.Running.DataDir = cliOpts.App.DataDir
+	}
+	err = createDataDir(cmdCtx.Running.DataDir)
```

Comment:
`if err = createDataDir(cmdCtx.Running.DataDir); err != nil {` Up to you.

Reply:
fixed